hs_core/hydroshare/users.py

```diff
@@ -238,19 +238,10 @@ def get_public_groups():
         return Group.objects.filter(gaccess__public=True)
 
 
-def get_resource_list(creator=None,
-        group=None, user=None, owner=None,
-        from_date=None, to_date=None,
-        start=None, count=None,
-        full_text_search=None,
-        published=False,
-        edit_permission=False,
-        public=False,
-        type=None,
-        author=None,
-        contributor=None,
-        subject=None,
-):
+def get_resource_list(creator=None, group=None, user=None, owner=None, from_date=None,
```

Comment:
@aphelionz  I am trying to find out how this list gets generated based on bounding box information.